### ISSUE TITLE
Use cf-deployment version as the helm chart appVersion

### ISSUE THREAD

**Issue by jandubois** [cloudfoundry-incubator/kubecf#926]
Currently we just repeat the regular `version` value in the `appVersion` field, which doesn't make much sense, given that `appVersion` is optional.

From https://helm.sh/docs/topics/charts/#the-appversion-field

>Note that the `appVersion` field is not related to the `version` field. It is a way of specifying the version of the application. For example, the `drupal` chart may have an `appVersion: 8.2.1`, indicating that the version of Drupal included in the chart (by default) is `8.2.1`. This field is informational, and has no impact on chart version calculations.

I think using the `cf-deployment` version makes the most sense for the `appVersion` field in our case.

**Comment by troytop** [comment]
I hereby I approve of this use of `appVersion` and recant my previous heresies that it should be used for the SUSE CAP product version.

**Comment by viovanov** [comment]
`cf-deployment` is just a dependency of kubecf, and getting smaller all the time, with a specific goal to deprecate it. We shouldn't publicize that version.

**Comment by jandubois** [comment]
> `cf-deployment` is just a dependency of kubecf

I disagree with that. It is the deployment manifest that is the base of the particular release. It pins the versions of all the bosh releases to to a particular set that have been published by upstream. I know we do override parts of them, but still, as long as upstream uses `cf-deployment` version numbers to specify a CFAR version, we might want to follow suit until something more appropriate comes along.

> getting smaller all the time

If we decide that `cf-deployment` version is not suitable for our needs, then I would suggest that we use the `capi-release` version number, as that explains the CC API version that we will support in that kubecf release.

**Comment by troytop** [comment]
`capi-release` works for me. It will continue to be relevant for the foreseeable future.
